snapcraft/storeapi/__init__.py

```diff
@@ -194,6 +194,31 @@ def release(self, snap_name, revision, channels):
         return self._refresh_if_necessary(
             self.sca.snap_release, snap_name, revision, channels)
 
+    def get_snap_status(self, snap_name, series=None, arch=None):
+        account_info = self.get_account_information()
+        snap_id = None
+        try:
+            if series:
+                snap_id = account_info['snaps'][series][snap_name]['snap-id']
+            else:
+                snap_id = None
+                for key, value in account_info['snaps'].items():
+                    if snap_name in value:
+                        snap_id = value[snap_name]['snap-id']
```

Comment:
It's possible if the snap name is registered in different series (`account_info['snaps']` is a dict with series name as keys), but it should be ok as they should all share the same snap id. I'm adding a `break` once found so we keep the 1st one.